packages/_shared/src/hooks/useActiveLocales.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+
+import { BaseAppSDK, SharedEditorSDK } from '@contentful/app-sdk';
+
+type LocaleSettings = {
+  mode: 'single' | 'multi';
+  active?: string[];
+  focused?: string;
```

Comment:
We could ignore single and focused here, as with the new translation widget is stays consistent :)